src/storage/file_io.c

```diff
@@ -8544,15 +8544,15 @@ fileio_flush_backup (THREAD_ENTRY * thread_p, FILEIO_BACKUP_SESSION * session_p)
 {
   char *buffer_p;
   ssize_t nbytes;
-  int count;
+  INT64 count;
   bool is_interactive_need_new = false;
   bool is_force_new_bkvol = false;
 
-  if ((int) prm_get_bigint_value (PRM_ID_IO_BACKUP_MAX_VOLUME_SIZE) > 0
-      && session_p->bkup.count > (int) prm_get_bigint_value (PRM_ID_IO_BACKUP_MAX_VOLUME_SIZE))
+  if (prm_get_bigint_value (PRM_ID_IO_BACKUP_MAX_VOLUME_SIZE) > 0
+      && session_p->bkup.count > prm_get_bigint_value (PRM_ID_IO_BACKUP_MAX_VOLUME_SIZE))
```

Comment:
It seems that the data type of session_p-> bkup.count should be changed to UINT64.

Reply:
OK. I accept your opinion, so modify.